modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,280 @@
+* You have installed the {gitops-title} CLI on your system. For more information, see xref:../installing_gitops/installing-argocd-gitops-cli.adoc#installing-argocd-gitops-cli[Installing the {gitops-shortname} CLI].
+* You have installed the Argo Rollouts CLI on your system. For more information, see _Argo Rollouts CLI overview_.
+
+.Procedure 
+
+. Create a `Route` object.
+.. In the *Administrator* perspective of the web console, click *Networking* -> *Routes*.
+.. Click *Create Route*.
+.. On the *Create Route* page, click *YAML view* and add the following snippet:
++
+The following example creates a route called `rollouts-demo-route`:
++
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rollouts-demo-route
+spec:
+  port:
+    targetPort: http #<1>
+  tls: #<2>
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: argo-rollouts-stable-service #<3>
+    weight: 100 # <4>
+
+  alternateBackends:
+    - kind: Service
+      name: argo-rollouts-canary-service #<5>
+      weight: 0 #<6>
+----
+<1> Specifies the name of the port used by the application for running inside the container.
+<2> Specifies the TLS configuration used to secure the route
+<3> The name of the targeted stable service.
+<4> This field is automatically modified to stable weight by Route Rollout plugin.
+<5> The name of the targeted canary service.
+<6> This field is automatically modified to canary weight by Route Rollout plugin.
++
+.. Click *Create* to create the route. It is then displayed on the *Routes* page.
+. Create the services, canary and stable, to be referenced in the route.
+.. In the *Administrator* perspective of the web console, click *Networking* -> *Services*.
+.. Click *Create Service*.
+.. On the *Create Service* page, click *YAML view* and add the following snippet:
++
+The following example creates a canary service called `argo-rollouts-canary-service`. Canary traffic is directed to this service.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: argo-rollouts-canary-service
+spec:
+  ports: #<1>
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: #<2>
+    app: rollouts-demo
+----
+<1> Specifies the name of the port used by the application for running inside the container.
+<2> Ensure that the contents of the `selector` field are the same as in stable service and `Rollout` CR.
++
+[IMPORTANT]
+====
+Ensure that the name of the canary service specified in the `Route` object matches with the name of the canary service specified in the `Service` object.
+====
+.. Click *Create* to create the canary service.
++
+Rollouts automatically update the created service with pod template hash of the canary `ReplicaSet`. For example, `rollouts-pod-template-hash: 7bf84f9696`.
+.. Similarly, repeat these steps to create the stable service:
```

Comment:
In my opinion "similarly" is unnecessary and gets in the way of the actual step being instructed:
```suggestion
.. Repeat these steps to create the stable service:
```

Reply:
Fixed.